src/utils/comma-separated-list.ts

```diff
@@ -26,3 +26,22 @@ export const CommaSeparatedListOf = (decoder: t.Mixed) =>
       ),
     String
   );
+
+export const ArbitrarySeparatedListOf = (separator: string, decoder: t.Mixed) =>
```

Comment:
If we rewrite this as 
```
export const GetArbitrarySeparatedListOf = (separator: string) => (decoder: t.Mixed) =>
```
we can reuse this Type generator to create different types decoder like
```
export const CommaSeparatedListOf = GetArbitrarySeparatedListOf(",");
export const PipeSeparatedListOf = GetArbitrarySeparatedListOf("|")
...
```